### PR TITLE
Added strided interface to fill a strided array with pseudorandom numbers drawn from a Laplace distribution.

### LINKED ISSUE #940: [RFC]: Add `@stdlib/random/strided/laplace`
### Description

This RFC proposes adding support for filling a strided array with pseudorandom numbers drawn from a Laplace distribution. This is the strided interface analog of `@stdlib/random/base/laplace`.

Package: `@stdlib/random/strided/laplace`
Alias: `laplace`

### Related Issues

None.

### Questions

No.

### Other

- A package with a similar implementation/API design is [`@stdlib/random/strided/beta`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/random/strided/beta).

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by ShivangRawat30**
…r of laplace

Resolves # .

## Description

> What is the purpose of this pull request?
Added strided interface to fill a strided array with pseudorandom numbers drawn from a Laplace distribution.

This pull request:

-   a
-   b
-   c

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #940
-   fixes #940

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [ ] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/random/strided/laplace/README.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Beta Random Numbers
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/laplace/README.md:21]:
```suggestion
# Laplace Random Numbers
```

File: lib/node_modules/@stdlib/random/strided/laplace/README.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+> Fill a strided array with pseudorandom numbers drawn from a [laplace][@stdlib/random/base/laplace] distribution.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/laplace/README.md:23]:
```suggestion
> Fill a strided array with pseudorandom numbers drawn from a [Laplace (double exponential)][@stdlib/random/base/laplace] distribution.
```

File: lib/node_modules/@stdlib/random/strided/laplace/README.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+## Usage
+
+```javascript
+var laplace = require( '@stdlib/random/strided/laplace' );
+```
+
+#### laplace( N, alpha, sa, beta, sb, out, so\[, options] )
+
+Fills a strided array with pseudorandom numbers drawn from a [laplace][@stdlib/random/base/laplace] distribution.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/laplace/README.md:35]:
```suggestion
Fills a strided array with pseudorandom numbers drawn from a [Laplace (double exponential)][@stdlib/random/base/laplace] distribution.
```

File: lib/node_modules/@stdlib/random/strided/laplace/package.json [hunk 4]
```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@stdlib/random/strided/laplace",
+  "version": "0.0.0",
+  "description": "Fill a strided array with pseudorandom numbers drawn from a beta distribution.",
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/laplace/package.json:4]:
```suggestion
  "description": "Fill a strided array with pseudorandom numbers drawn from a Laplace (double exponential) distribution.",
```

File: lib/node_modules/@stdlib/random/strided/laplace/package.json [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    "mathematics",
+    "math",
+    "statistics",
+    "stats",
+    "prng",
+    "rng",
+    "pseudorandom",
+    "random",
+    "rand",
+    "beta",
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/laplace/package.json:63]:
```suggestion
    "laplace",
```

File: lib/node_modules/@stdlib/random/strided/laplace/README.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var laplace = require( '@stdlib/random/strided/laplace' );
+```
+
+#### laplace( N, alpha, sa, beta, sb, out, so\[, options] )
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/laplace/README.md:33]: Copy-paste error: Need to update distribution parameters here and elsewhere in the `README.md` from `alpha` and `beta` to `mu` (mean) and `b` (scale parameter) as in implementation.

**Review by Planeshifter — CHANGES_REQUESTED** [review]
Thank you for your contribution!

There are a few documentation and copy-paste errors that need to be resolved before merging.

**Comment by ShivangRawat30** [comment]
@Planeshifter Thanks for the input, I'll change them asap

**Comment by Planeshifter** [comment]
Closing again as superseded by https://github.com/stdlib-js/stdlib/pull/1011.

**Pull request closed without merging**